utils/uf2conv.py

```diff
@@ -238,6 +211,17 @@ def write_file(name, buf):
     print("Wrote %d bytes to %s" % (len(buf), name))
 
 
+def load_families():
+    with open("uf2families.json") as f:
```

Comment:
where should we look for this file? I think this checks the current folder, while I think it would be wiser to look in the same place where the script is placed?

Reply:
The current JSON file is in that location. I can add code that will retrieve it from a fixed URL if you like, but right now the JSON file is in the `utils` directory with this script.

Reply:
I think that if you run:

```
cd ~/myrepo
python uf2/utils/uf2conv.py foo.bin
```

then it will look for the families in `~/myrepo` but it should look in `~/myrepo/uf2/utils`. People often include uf2 as submodule just for that script. 

No need to download.

Reply:
Understood. I have made a change that I think addresses this concern.

Reply:
Changed it to your version.